src/main/java/at/hannibal2/skyhanni/features/misc/items/EstimatedItemValueTooltip.kt

```diff
@@ -0,0 +1,24 @@
+package at.hannibal2.skyhanni.features.misc.items
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.api.event.HandleEvent
+import at.hannibal2.skyhanni.events.item.ItemHoverEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.ItemPriceUtils.getPrice
+import at.hannibal2.skyhanni.utils.ItemUtils.getInternalNameOrNull
+import at.hannibal2.skyhanni.utils.NumberUtil.addSeparators
+
+@SkyHanniModule
+object EstimatedItemValueTooltip {
+
+    @HandleEvent(onlyOnSkyblock = true)
+    fun onTooltip(event: ItemHoverEvent) {
+        if (!SkyHanniMod.feature.inventory.estimatedItemValues.showTooltip) return
+        val internalName = event.itemStack.getInternalNameOrNull() ?: return
+
+        val (total, _) = EstimatedItemValueCalculator.calculate(event.itemStack, mutableListOf())
+        if (total == 0.0) return
```

Comment:
the check for 0 is not necessary. there should be no case where this helps us. if there are cases, we would also see the estimated item value show up for such items as we nowhere else in the estimated item value code check against 0